cartographer/common/CMakeLists.txt

```diff
@@ -18,195 +18,32 @@ configure_file (
   ${CMAKE_CURRENT_SOURCE_DIR}/config.h.cmake
```

Comment:
Remove the `add_subdirectory("proto")` together with the CMakeLists.txt in there? Same in other cases.